### ISSUE TITLE
Page 181 - FORGE-2 Subaltern Squad Wording Discrepancy/Clarification

### ISSUE THREAD

**Issue by KaiTave** [RancidRance/Wallflower-Part-1-Release-Typos-and-Errors#150]
The way the Kidd's FORGE-2 Subaltern Squad is worded, several of its abilities only affect "allied characters," which by strict Rules As Written does not include the mech which deployed them. At one point the Forge Subalterns were drones, and therefore "allied characters" would be from the perspective of the drones and include the user as well, but deployables aren't characters and so the wording may lead people to conclude the user can't benefit from the system's own effects.

I don't believe this is an intended effect, and if it isn't then it may be helpful to clarify the wording in a couple of places just to make it more obvious.

"DAMPING STATION : When finished, this repair station generates a BURST 1 field within which allied characters, **including yourself,** gain IMMUNITY to burn and clear any burn currently them."

"ARMOR PACK : When finished, this pack latches onto the next allied character, **including yourself,** that moves adjacent to it and fuses onto them."

**Comment by Piezoelectric** [comment]
I personally might prefer 

> "...within which yourself or an allied character gains IMMUNITY to burn..."

> "...latches onto the next character, either yourself or an allied character, that moves adjacent..."

So that it's explicitly clear that "yourself" is considered separate from "allied character", instead of a modifier/qualifier on "allied character"

**Comment by KaiTave** [comment]
Yeah those wordings are honestly better imo, I'd lean towards them.
